api/login.go

```diff
@@ -25,22 +28,29 @@ func (cli *Client) LoginWithGithub(endpoint string, githubToken string) (*User,
 	return userResp.User, nil
 }
 
-func (cli *Client) LoginWithVault(endpoint string, vaultToken string) (*User, error) {
-	url := endpoint + pathPrefix + "/login"
-	req, err := http.NewRequest("POST", url, nil)
+func (cli *Client) LoginWithVault(endpoint string, vault_url string, githubToken string) (*User, error) {
+
+	req, err := http.NewRequest("POST",
+		vault_url+"/v1/auth/github/login",
```

Comment:
I think it is better to crate a variable like you do in `LoginWithGithub` function.

I assume that you can do something like
```
		vault_url+ pathPrefix +"/auth/github/login",
```

Reply:
Path prefix is not required for vault... I don't know what to put here.